scripts/test262-runner.js

```diff
@@ -397,11 +397,10 @@ function workerArgsParse(): WorkerProgramArgs {
 function masterRun(args: MasterProgramArgs) {
   let testPath = `${__dirname}` + args.relativeTestPath + "/test";
   let tests = getFilesSync(testPath);
-  // remove tests that don't need to be ran
+  // remove tests that don't need to be run
+  if (args.filterString) tests = tests.filter(test => test.location.includes(args.filterString));
```

Comment:
I don't want to count in all of the tests that are not selected by the filterString since that is exclusively used to reduce the number of tests to run while debugging.